.github/workflows/cut-new-release.yml

```diff
@@ -75,20 +75,20 @@ jobs:
     needs: [create-new-version]
     uses: ./.github/workflows/build.yml
     with:
-      ref: ${{ needs.create-new-version.outputs.version }}
+      ref: ${{ jobs.create-new-version.outputs.version }}
```

Comment:
It won't work unfortunately. According to https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/passing-information-between-jobs the job needs to be listed in `need` to get access to its output.

Reply:
I had to revert it in 54936838a3b4b999db3183a1345ef7031517cdef. Tested that indeed it won't work in https://github.com/ifrost/deployme/actions/runs/13291199875